### DISCUSSION TITLE
각종 등재 사이트

### DISCUSSION THREAD

**Discussion by gintering1000** [orioncactus/pretendard#53] [Ideas]
소스단위
- Github or BitBucket

결과물 단위 : ttf, otf
- 공식사이트
- Google Font
- 눈누

운영체제 단위
- MacOS : brew
- Windows : chocolaty
- Ubuntu : apt-get
- Centos : yum

CDN 단위
- jsDelivr
- cdnjs

웹개발 단위
- npm
- yarn

프로그래밍 언어단위 ( 비추 )
- pip ( python )
- composer ( php )

위에서부터 하나~씩 차근차근 해나가시면 될듯합니다

이정도 퀄리티에 완벽함이라면 콘솔용 mono, 에코 정도만 추가되면 10~20년은 글로벌 1위겠네요
( 조만간 애플에서 연락올겁니다. 별거아닌것 같지만 9가지로 세분화된 제작용 폰트 애플말고 본적이 없습니다. 제작자들에게 상당히 귀합니다 )
하지만 숙제는 등재 ( 등재가 되어야 사용자들이 많이 쓰고, 많이 써야 문제점이 하나둘 들어납니다 )

**Comment by orioncactus** [comment]
안녕하세요! 이렇게 관심을 가져주시고 여러 군데로부터 Pretendard를 효율적으로 사용할 수 있게 리스트를 정리해주셔서 진심으로 감사드립니다. 말씀해주신 것처럼 차근차근 준비해 더 많은 사람들이 편리하게 Pretendard를 쓸 수 있도록 나아가려고 합니다 ☺️

**Comment by kms0219kms** [comment]
안녕하세요! Pretendard를 배포할 수 있는 플랫폼 리스트들을 적어주셔서 진심으로 감사드립니다.
지금으로서 할 수 있는 부분만 먼저 등재를 진행해 보면 어떨까 싶습니다.

아래 부분은 To-Do 형식으로 리스트를 정리 해둔것입니다. 참고 해 주시면 감사하겠습니다!

소스단위
- [x] Github

결과물 단위 : ttf, otf
- [x] 공식 사이트
- [ ] Google Fonts
- [x] 눈누(배포방식 변경 논의 중)

운영체제 단위
- [x] MacOS : brew, dmg-autoinstall(실효성 확인 중)
- [ ] Windows : ~~chocolatey~~ -> exe-autoinstall(실효성 확인 중)
- [ ] Ubuntu : apt-get
- [ ] Centos : yum

CDN 단위
- [x] jsDelivr
- [x] cdnjs
- [x] unpkg

프로그래밍 언어단위
- [x] npm ( node.js )
- [x] yarn ( node.js )
- [ ] ~~pip ( python )~~
- [ ] ~~composer ( php )~~

**Reply by orioncactus** [reply]
@sudosubin님의 기여 덕분에 homebrew-cask-fonts에도 등재되었습니다! 🥺
- https://github.com/Homebrew/homebrew-cask-fonts/pull/5991

**Comment by kms0219kms** [comment]
여담으로, 윈도우/맥 용 폰트 자동 설치프로그램 들도 있으면 사용하기 편할거 같네요!
위 체크박스에 추가해두도록 하겠습니다

**Reply by orioncactus** [reply]
체크리스트를 업데이트해주셔서 감사드립니다! 다만 설치 프로그램을 통한 글꼴 설치는 실효성에 의문이 있습니다. macOS나 Windows 모두 otf나 ttf를 통한 글꼴 파일 설치 과정이 몇 번의 클릭이면 되기 때문에, 꼭 자동 설치 프로그램을 만들지는 않아도 될 듯합니다. 또한 Pretendard는 운영체제별로 확장자를 나눠 만들지 않았기 때문에(Windows에서 가독성을 고려하면 ttf를 써야하긴 하지만요) 2개의 운영체제에서 각각 2개의 설치 프로그램을 관리하기에는 어려움이 있지 않을까 싶어서, 추후 필요해질 때 고려해도 괜찮을 것 같습니다 👀

**Reply by kms0219kms** [reply]
단 현재 zip 파일의 구조가 복잡하여, 다운로드 후 해당 경로까지를 들어가서 설치해야 하다보니 다른 폰트보다 설치가 복잡하다는 단점이 있습니다. 그렇다보니 자동 설치를 만드는게 어떨까라는 생각을 하게 된거구요.

**Reply by kms0219kms** [reply]
또한 가장 최신릴리즈를 가리키는 깃허브 경로만 있다면 추가적인 유지보수는 크게 필요가 없을수도 있습니다.
